### ISSUE TITLE
[Bug]: `ERROR: Invalid syntax. Type "REG ADD /?" for usage.`

### ISSUE THREAD

**Issue by guihkx** [undergroundwires/privacy.sexy#380]
### Description

When applying the `Disable Microsoft feature trials` tweak from version 0.13.5, I got the following error:

```
--- Disable Microsoft feature trials
The operation completed successfully.
The operation completed successfully.
ERROR: Invalid syntax.
Type "REG ADD /?" for usage.
```

### How can the bug be recreated?

1. Apply the `Disable Microsoft feature trials` tweak

### Operating system

Windows 11 Pro 24H2 (build 26100.1000)

**Comment by undergroundwires** [comment]
Thanks for the bug report @guihkx.

It's really helpful with all details you provide such as OS/app version.

I see that it was introduced as part of 0.13.4 due to a typo included in [commit cec0b4b](https://github.com/undergroundwires/privacy.sexy/commit/cec0b4b4f63c3563a0e7923ce6324a38d71a3955#diff-cc6832e8ee0b6b22d33fa5081a4ca1a4ffebf24d56d590d2fc52886a5025e9bfR4750). It's due to `/f` flag incorrectly being `/`.

I will fix this by getting rid of all these manual `reg add` commands by replacing them with a common function like `SetRegistryValue` in [`windows.yaml`](https://raw.githubusercontent.com/undergroundwires/privacy.sexy/0.13.5/src/application/collections/windows.yaml). This way, the change will not only fix this bug but will prevent the code against this kind of typos again.

**Comment by guihkx** [comment]
Superb! Thank you for looking into this so quickly. :)
